test/rebuild.ts

```diff
@@ -144,6 +144,34 @@ describe('rebuilder', () => {
       expect(built).to.equal(1);
     });
 
+    it('should rebuild only specified modules with prefixes', async () => {
+      const rebuilder = rebuild({
+        buildPath: testModulePath,
+        electronVersion: '5.0.13',
+        arch: process.arch,
+        onlyModules: ['@nlv8/signun'],
+        force: true
+      });
+      let built = 0;
+      rebuilder.lifecycle.on('module-done', () => built++);
+      await rebuilder;
+      expect(built).to.equal(1);
+    });
+
+    it('should rebuild only specified modules missing prefixes', async () => {
+      const rebuilder = rebuild({
+        buildPath: testModulePath,
+        electronVersion: '5.0.13',
+        arch: process.arch,
+        onlyModules: ['signun'],
+        force: true
+      });
+      let built = 0;
+      rebuilder.lifecycle.on('module-done', () => built++);
+      await rebuilder;
+      expect(built).to.equal(1);
+    });
```

Comment:
I think this behavior should be dropped in version 2 - @electron/wg-ecosystem does anyone have differing opinions on this, or know of any legitimate use cases for this behavior?